frontend/app/chat/components/ChatsList/components/ChatsListItem/components/UserButton.tsx

```diff
@@ -1,13 +1,17 @@
 import Link from "next/link";
 import { MdPerson } from "react-icons/md";
 
+import { useUserData } from "@/lib/hooks/useUserData";
+
 import { sidebarLinkStyle } from "../styles/SidebarLinkStyle";
 
 export const UserButton = (): JSX.Element => {
+  const { userData } = useUserData();
+
   return (
     <Link aria-label="account" className={sidebarLinkStyle} href={"/user"}>
       <MdPerson className="text-4xl" />
-      <span>Account</span>
+      <span>{userData?.email}</span>
```

Comment:
If userData is undefined so will the email. It will appear as "undefined" in the front. Maybe change it `userData?.email ?? ''` so that its empty when undefined (e.g before fetching data from backend)

Reply:
Ok ! I took it into account in the next commit ([5c7b365](https://github.com/StanGirard/quivr/pull/1262/commits/5c7b36537e6d961c8cd38e2e2a0b84cdd867085c))